suites/nautilus/rgw/dynamic_resharding_greenfield.yaml

```diff
@@ -0,0 +1,127 @@
+# Verify dynamic resharding feature on multisite
```

Comment:
Could we rename the file to `tier-1_rgw_test-dynamic-resharding-greenfield.yaml`? This would enable us to follow the naming guidelines put forth.